ee-installer/src/book-assets/README.md

```diff
@@ -0,0 +1,18 @@
+# Book Assets
```

Comment:
Please don't duplicate files. Instead symlink any asset directories (or files) to the ones in the root book. mdbook build follows symlinks

Reply:
At least symlinks works locally, hopefully this works with GitHub actions as well

Reply:
Thankfully, they work with GitHub actions as well and I have now removed the files and symlinked the directory.